pkg/migrations/op_add_column_test.go

```diff
@@ -150,7 +150,25 @@ func TestAddColumn(t *testing.T) {
 					},
 				},
 			},
-			wantStartErr: backfill.NotPossibleError{Table: "users"},
+			afterRollback: func(t *testing.T, db *sql.DB, schema string) {
```

Comment:
I think the test should have an `afterStart` part too - even if it's very simple.